docs/docs/unreachable.md

```diff
@@ -0,0 +1,143 @@
+
+---
+title: Unreachable Strategy
+---
+
+# Unreachable Strategy
+
+In order for Marathon to provide partition aware unreachable strategy support there are 2 high level events that must occur:
+
+1. Mesos needs to communicate a task is unreachable
+2. Marathon must respond to that event if unresolved within a specified amount of time.
+
+Each of these events have configuration options and DC/OS system defaults which are worth review in order to fully understand how and when an unreachable task will be managed by Marathon.
+
+## Apache Mesos Unreachable Strategies
+
+### Inactive Agent Logic and Unreachable Tasks
+
+Apache Mesos ability to communicate a task / node is unreachable is controlled by 2 concepts:
+
+1. Mesos agent health checks.
+2. Node rate limiter.
+
+Regarding agent health checks, the Mesos master flags of control are:
+
+- `--agent_ping_timeout` (default 5) - The duration after which an attempt to ping the agent is considered a timeout
+- `--max_agent_ping_timeouts` (default 15s) - The number of consecutive attempts to ping an agent, after which the agent is considered unreachable.
+
+
+The Mesos defaults (of 15s \* 5 timeouts) effectively will cause the Mesos master to consider an agent "inactive" after a 75 seconds of no communication (assuming the loss of 1 agent).
```

Comment:
>  communication (assuming the loss of 1 agent).

I'm probably ruin for life here but one of my professors had quite some arguments against using parenthesis. What are you trying to express? I'd just leave them.

Reply:
✅